ai_edge_torch/generative/test/test_model_conversion.py

```diff
@@ -55,6 +55,30 @@ def test_toy_model_with_kv_cache(self):
           )
       )
 
+  def test_toy_model_with_multi_batches(self):
```

Comment:
Nit: Please feel free to ignore it for this PR, but some of these tests can be combined and parametrized to avoid duplicated code.